api/src/pcapi/routes/backoffice/offers/blueprint.py

```diff
@@ -181,7 +182,7 @@ def _get_offer_ids_query(form: forms.InternalSearchForm) -> BaseQuery:
             operators_definition=OPERATOR_DICT,
         )
         for warning in warnings:
-            flash(warning, "warning")
+            flash(Markup("{message}").format(message=warning), "warning")
```

Comment:
[détail / question de goût] J'ai failli proposer de simplement écrire `flask(Markup(warning), "warning")`, mais c'était un piège, ça ne fait pas ce que l'on veut : 

```python
>>> s = "<hey>"
>>> Markup(s)
Markup('<hey>')   # oups, les "<" et ">" ne sont pas échappés !
>>> Markup('{s}').format(s=s)
Markup('&lt;hey&gt;')
```

Par contre, on peut écrire : 

```suggestion
            flash(markupsafe.escape(warning), "warning")
```

Reply:
J'avais finalement remplacé les `Markup("{message}").format(message=warning)` par `escape(warning)` ailleurs dans la PR, j'ai loupé celui-là, merci !